dapps/ui-deprecation/build/index.html

```diff
@@ -107,7 +107,7 @@
 		<div class="parity-navbar">
 		</div>
 		<div class="parity-box">
-			<h1>Parity Wallet is deprecated.</h1>
+			<h1>Parity browser UI is deprecated.</h1>
```

Comment:
I thought the product name is actually `Parity Wallet`, no?

Reply:
We are moving in circles here :rofl: 

https://github.com/paritytech/parity/pull/8164#issuecomment-374622582

Reply:
Yeah, afaiu we are moving away from "Parity Wallet" (browser wallet) in favour of standalone "Parity UI" (electron app).

Reply:
It is, Parity will not support any dapps, UI, wallets whatsoever. The distinction between "Parity Wallet" being a dapp was not really communicated I don't recall changing the names when UI2 has been rolled out. That said I'm not really requesting a change here, just discussing if that's a correct "story" we are telling here.

Anyway, looks good to me :+1: :)